index.js

```diff
@@ -120,6 +120,10 @@ function ui () {
   print()
 
   if (allowedPeers !== undefined) {
+    const swarmPublicKey = lists[0].userData.swarmPublicKey
+    if (swarmPublicKey) {
+      print('Seedbee writer:', crayon.green(HypercoreId.encode(swarmPublicKey)))
```

Comment:
Is it the writer? Probably more like `Seedbee swarm`

Reply:
It could also be more generic like "Frontend swarm" so it's easier to know where it is